apps/backend/src/data/migrations/1654703454488-create-category-modifiers-table.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey
+} from 'typeorm';
+
+export class CreateCategoryModifiersTable1654703454488
+  implements MigrationInterface
+{
+  async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'modifier-category',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment'
+          },
+          {
+            name: 'name',
+            type: 'varchar'
```

Comment:
please specify length for each varchar/text type
for each object/column you can specify `length` property with the varchar/text maximum length